src/main/java/randoop/reflection/OperationExtractor.java

```diff
@@ -152,6 +182,13 @@ public void visit(Constructor<?> constructor) {
     TypedClassOperation operation = instantiateTypes(TypedOperation.forConstructor(constructor));
     checkSubTypes(operation);
     if (!omitPredicate.shouldOmit(operation)) {
+      if (operationSpecifications != null) {
+        OperationConditions conditions =
+            operationSpecifications.getOperationConditions(constructor);
```

Comment:
Because the type hierarchy is a DAG, it looks like this may add duplicates to `operationSpecifications` when a superclass is visited more than once.  In a comment, discuss whether this is true, and what the consequences of it are if it is true.